CMakeLists.txt

```diff
@@ -297,7 +297,11 @@ set(EXPAT_BUILD_DOCS OFF)
 set(EXPAT_BUILD_PKGCONFIG OFF)
 set(EXPAT_ENABLE_INSTALL OFF)
 set(EXPAT_SHARED_LIBS OFF)
-add_subdirectory(3party/expat/expat)
+find_package_or_fallback_to_3party(
+  "expat"
+  "2.5.0"
+  "3party/expat/expat")
```

Comment:
nit: do it in one line. Fewer lines = easier to read/overview code.

Reply:
Done